src/pages/CategoriesPage/CategoriesPageContainer.tsx

```diff
@@ -0,0 +1,15 @@
+import { useAppSelector } from '../../app/hooks';
+import { RootState } from '../../app/store';
+import { CategoryModel } from '../../models/CategoryModel';
+import CategoriesPageComponent from './CategoriesPageComponent';
+
+export const CategoriesPageContainer = () => {
+  const categories: CategoryModel[] = useAppSelector((state: RootState) =>
+    state.catalog.categoryIds.map((id) => state.catalog.categories[id as number - 1])
```

Comment:
Why -1?

Reply:
Slice logic has been changed, so this code has been cleaned  up from the decrementing

Reply:
I need to create new Id for my category in the code, so I decided to implement the logic in this way. To my mind, this approach maybe looks like naively, but I tried to generate new Id using my CategoryIds array